dev_guide/pod_autoscaling.adoc

```diff
@@ -129,14 +126,13 @@ spec:
   maxReplicas: 10 <6>
   targetCPUUtilizationPercentage: 80 <7>
 ----
-<1> The name of this horizontal pod autoscaler object
-<2> The kind of object to scale
-<3> The name of the object to scale
-<4> The API version of the object to scale
-<5> The minimum number of replicas to which to scale down
-<6> The maximum number of replicas to which to scale up
-<7> The percentage of the requested CPU that each pod should ideally be using
-====
+<1> The name of this horizontal pod autoscaler object.
+<2> The kind of object to scale.
+<3> The name of an exisiting object you want to scale.
+<4> The API version of the object to scale.
+<5> The minimum number of replicas to which to scale down. The default is `1`.
```

Comment:
to which to scale down -> to scale down

Reply:
I found better wording in the 4.x docs:
_The minimum number of replicas when scaling down._
_The maximum number of replicas when scaling up._